app/src/pages/inside/membersPage/membersGrid/projectRole/projectRole.jsx

```diff
@@ -0,0 +1,125 @@
+  activeProjectSelector,
+  activeProjectRoleSelector,
+  accountRoleSelector,
+  isAdminSelector,
+} from 'controllers/user';
+import { ROLES_MAP, ADMINISTRATOR } from 'common/constants/projectRoles';
+import { showNotification } from 'controllers/notification';
+
+import styles from './projectRole.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
+  projectRoleMobile: { id: 'ProjectRole.mobileTitle', defaultMessage: 'Project role' },
+  allPermissions: {
+    id: 'ProjectRole.allPermissions',
+    defaultMessage: 'Administrator has full privileges according to permission map',
+  },
+  updateMember: {
+    id: 'ProjectRole.updateMember',
+    defaultMessage: 'Member {name} has been updated',
+  },
+});
+
+@injectIntl
+@connect(
+  (state) => ({
+    projectId: activeProjectSelector(state),
+    isAdmin: isAdminSelector(state),
+    canChangeRole: () =>
+      canChangeUserRole(accountRoleSelector(state), activeProjectRoleSelector(state)),
+  }),
+  { showNotification },
+)
+export class ProjectRole extends PureComponent {
+  static propTypes = {
+    assignedProjects: PropTypes.object,
+    accountRole: PropTypes.string,
+    projectId: PropTypes.string,
+    intl: intlShape.isRequired,
+    userId: PropTypes.string,
+    showNotification: PropTypes.func,
+    canChangeRole: PropTypes.func,
+    isAdmin: PropTypes.bool,
+  };
+  static defaultProps = {
+    assignedProjects: {},
+    accountRole: '',
+    projectId: '',
+    userId: '',
+    showNotification: () => {},
+    canChangeRole: () => {},
+    isAdmin: false,
+  };
+  state = {
+    currentRole: this.getUserRole(),
+  };
+
+  onChangeRole = (val) => {
+    const { intl, userId } = this.props;
+    const param = {
+      users: {},
+    };
+    param.users[this.props.userId] = val;
+    this.setState({ currentRole: val });
+    fetch(URLS.project(this.props.projectId), {
+      method: 'put',
+      data: param,
+    })
+      .then(() => {
+        this.props.showNotification({
+          message: intl.formatMessage(messages.updateMember, { name: userId }),
+          type: 'success',
+        });
+      })
+      .catch((err) => {
+        this.props.showNotification({
+          message: err.msg,
+          type: 'error',
+        });
+      });
+  };
+  getUserRole() {
+    const { assignedProjects, projectId } = this.props;
+    return assignedProjects[projectId].projectRole;
+  }
+  render() {
+    this.getUserRole();
+    const { accountRole, intl } = this.props;
+    return (
+      <Fragment>
+        <span className={cx('mobile-title', 'mobile-show')}>
+          {intl.formatMessage(messages.projectRoleMobile)}:
+        </span>
+        <span className={cx('current-role', 'mobile-show')}>
+          {accountRole === ADMINISTRATOR
+            ? intl.formatMessage(messages.allPermissions)
+            : this.state.currentRole}
+        </span>
+        <div className={cx('roles-list', 'mobile-hide', { hide: accountRole === ADMINISTRATOR })}>
```

Comment:
I think if you not admin it's better to not render component ( conditional rendering ) instead of hiding it by className adding